code/modules/mob/living/living.dm

```diff
@@ -49,20 +49,61 @@
 		message = FALSE
 	return ..()
 
+// MOJAVE SUN EDIT BEGIN
 /mob/living/proc/ZImpactDamage(turf/T, levels)
 	if(SEND_SIGNAL(src, COMSIG_LIVING_Z_IMPACT, levels, T) & NO_Z_IMPACT_DAMAGE)
 		return
+	var/mob/living/carbon/human/savage = src // Power armour landing interactions - Fall damage negation / general chaos creation
+	if(istype(savage.wear_suit, /obj/item/clothing/suit/space/hardsuit/ms13/power_armor)) // If they're wearing PA, it's time to go sicko mode and start doing some crazy shit.
```

Comment:
USE A SIGNAL CHUD!!!!!!!!!!!!!

Reply:
like... a traffic signal...?

Reply:
I don't have time to do it before the test and I'll fix it later :)

Reply:
Will need to refactor this.  Will merge for now for the tests.